docs/development/license-validation/importing-licenses.md

```diff
@@ -0,0 +1,65 @@
+<!--
+    This source file is part of the open source project
+    ExpressionEngine User Guide (https://github.com/ExpressionEngine/ExpressionEngine-User-Guide)
+
+    @link      https://expressionengine.com/
+    @copyright Copyright (c) 2003-2024, Packet Tide, LLC (https://packettide.com)
+    @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
+-->
+
+# Importing a License via the API
+
+[TOC]
+
+Importing a license into our system allows add-on developers to ensure that users who purchased their add-ons outside of our platform can validate their licenses. This documentation outlines the process for add-on developers to import licenses into our system.
+
+## Requesting API Key
+
+Before importing licenses, add-on developers must contact us to request an API key. The API key is necessary to authenticate requests made to our system. Once the API key is obtained, developers can proceed with the import process.
+
+## API Endpoint
+
+Add-on developers can import licenses by making a POST request to the following API endpoint:
```

Comment:
@matthewjohns0n I think we should mention that it has to be JSON here as well.

Reply:
Yeah I agree, it would be good to mention that.